src/CacheProfiles/CacheAllSuccessfulGetRequests.php

```diff
@@ -38,4 +38,14 @@ public function shouldCacheResponse(Response $response)
     {
         return $response->isSuccessful() || $response->isRedirection();
     }
+
+    /**
+     * @param \Illuminate\Http\Request $request
+     *
+     * @return bool
+     */
+    public function shouldInvalidateCache(Request $request)
+    {
+        return $request->isMethod('patch') || $request->isMethod('delete') || $request->isMethod('post');
```

Comment:
Hmm... the reason I'm doing this is because this way, I have control over the verbs I want to handle. There are other HTTP verbs which we don't handle.